docs/gettingstarted.md

```diff
@@ -0,0 +1,137 @@
+# Getting started
+
+---
+
+## Table of Contents
+
+ - [Installation and environment set up](#installation-and-environment-set-up)
+ - [Jupyter Notebooks](#jupyter-notebooks)
+ - [First model](#first-model)
+ - [Elements of the model](#elements-of-the-model)
+
+---
+
+
+## Installation and environment set up
+
+Please note that for development, we suggest using [Poetry](https://python-poetry.org/) for dependency management and packaging. Instructions for setting up the Python environment with Poetry are in the section [Reference](/reference/#set-up-environment-with-poetry).
```

Comment:
We recommend using Poetry for development of radCAD itself, but this shouldn't be front and center for getting started, where users just need to know how to install radCAD, which in the README is as simple as `pip install radCAD`.